ArduCopter/mode_auto.cpp

```diff
@@ -1669,7 +1669,13 @@ bool Copter::ModeAuto::verify_loiter_time()
     }
 
     // check if loiter timer has run out
-    return (((millis() - loiter_time) / 1000) >= loiter_time_max);
+    if (((millis() - loiter_time) / 1000) >= loiter_time_max) {
+        gcs().send_text(MAV_SEVERITY_INFO, "Reached command #%i",cmd.index);
+        return true;
+    } else {
```

Comment:
I'd typically do an early-return instead if the time hasn't been reached.

Reply:
Done. Spline, regular waypoint has it that way. Do I submit a PR for that?

Reply:
Submitted a PR #10630